src/main/java/pl/nogacz/chess/application/ChessNotation.java

```diff
@@ -12,6 +12,7 @@
  */
 public class ChessNotation implements Serializable {
     private static List<String> movesList = new ArrayList<>();
+    private static List<PawnClass> kicked= new ArrayList<PawnClass>();

```

Comment:
fix spacing
```kicked = new```

Reply:
kıcked kelimesi ile ne kastediyorsun?

Reply:
Yenilen taştan bahsediyorum hocam ChessNotation sınıfında isKisked şeklinde boolean olduğu için ben de o şekilde kullandım